.github/ISSUE_TEMPLATE/01-bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: If something isn't working as expected
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening an issue, make sure to check if the issue was not previously reported or fixed.
```

Comment:
```suggestion
        Before opening an issue, please make sure to check if the [issue tracker](https://github.com/Retrospring/retrospring/issues?q=is%3Aissue+label%3Abug) in case it has been previously reported or fixed.
```

Reply:
Same thing as with the other one for the feature request template.